coding-guides/starboard.md

```diff
@@ -0,0 +1,198 @@
+# Starboard
+
+This is a long awaited feature, requested by many people.
+
+Let's begin by talking about what a starboard is. This is an example taken from the Discord.js Official Server.
+
+![Starboard](/assets/starboard.png)
+
+A starboard is a popular feature in bots that serve as a channel of messages that users of the server find funny, stupid, or both! To make a functioning starboard, we need to monitor for a reaction being added to a message, and we'll do this with the `messageReactionAdd` and `messageReactionRemove` events.
+
+> Before we start, there is one thing you need to know. This code is compatible with only Guidebot Class. This is due to a restriction with passing `client` to the messageReactionAdd and messageReactionRemove events.
+
+So, let's begin!
+
+In this block, we just do some simple setup for later on. For ease, I personally define the message object as `message`, but this is completely optional. Next, we grab the starboardChannel key from the guilds settings. 
+
+Then, we preform a couple of checks on the reaction and the message. First, we check if the reaction is **NOT** the unicode star emote. Next, we preform two checks on the message, checking to see if the user who added the reaction authored the message, and if the message author is a bot. If neither of these checks return true, we're good to move on.
+
+```js
+module.exports = class {
+  constructor(client) {
+    this.client = client;
+  }
+
+  async run(reaction, user) {
+    const message = reaction.message;
+    if (reaction.emoji.name !== '⭐') return;
+    if (message.author.id === user.id) return message.channel.send(`${user}, you cannot star your own messages.`);
+    if (message.author.bot) return message.channel.send(`${user}, you cannot star bot messages.`);
+    const { starboardChannel } = this.client.settings.get(message.guild.id);
+  }
+}
+```
+
+Now, this next block may look real complex, but it really isn't. Let's break it down.
+
+First, we declare 6 variables here. `fetch` will fetch the last 100 messages in the starboard channel, `stars` attempts to find an embed who's footer ends with the message's id.
```

Comment:
mention what the 6 variables are, this will allow a user to _connect the dots_ so to speak